cmd/main.go

```diff
@@ -39,11 +39,8 @@ func getConfigMetadata() map[string][]config.Parameter {
 			config.Parameter{Name: "logFile", Tag: "", Default: "/dev/stderr", Validators: []config.Validator{}},
 			config.Parameter{Name: "logLevel", Tag: "", Default: "INFO", Validators: []config.Validator{config.StringOptionsValidatorFactory([]string{"DEBUG", "INFO", "WARNING", "ERROR"})}},
 		},
-		"amqp1": []config.Parameter{
```

Comment:
Why delete this? We can let users decide whether they use bridge or straight access to bus.